tests/docker/Dockerfile

```diff
@@ -54,7 +62,7 @@ RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFK
     make install && \
     cd /opt/librdkafka/tests && \
     make build -j$(nproc) && \
-    go get -u github.com/birdayz/kaf/cmd/kaf && \
+    go get github.com/birdayz/kaf/cmd/kaf && \
```

Comment:
With the updated golang version, there were installation issues with one of kaf's dependencies:
```
go get: github.com/golang/lint@v0.0.0-20181026193005-c67002cb31c3 updating to
	github.com/golang/lint@v0.0.0-20210508222113-6edffad5e616: parsing go.mod:
	module declares its path as: golang.org/x/lint
	        but was required as: github.com/golang/lint
```

Apparently, others are having the same issue https://github.com/birdayz/kaf/issues/145

Removing the `-u` flag fixed the problem and all tests (ducktape and unit tests) passed.